src/views/WafRules/FormFields/FormFieldsWafRules.vue

```diff
@@ -51,12 +51,24 @@
   >
     <template #inputs>
       <div class="flex flex-col sm:max-w-lg w-full gap-2">
-        <FieldText
-          label="Name *"
-          name="Give a unique and descriptive name to identify the WAF rule."
```

Comment:
name="Give a unique and descriptive name to identify the WAF rule."
parece que o problema está aqui:
:name="name"
:placeholder="Give a unique and descriptive name to identify the WAF rule."

Colocar novamente o Field Text ao invés do label+InputText+smalls

Reply:
o Field Text usa o name, para via hook acessar o valor. Parece ter sido colocado o placeholder no lugar do name.

Reply:
Eu não sei se é um placeholder ou um descrição